src/palace/manager/sqlalchemy/model/time_tracking.py

```diff
@@ -74,12 +74,15 @@ class PlaytimeEntry(Base):
     collection: Mapped[Collection] = relationship("Collection", uselist=False)
     library: Mapped[Library] = relationship("Library", uselist=False)
 
+    loan_identifier = Column(String(50), nullable=False)
+
     __table_args__ = (
         UniqueConstraint(
             "tracking_id",
             "identifier_str",
             "collection_name",
             "library_name",
+            "loan_identifier",
```

Comment:
I'm not entirely sure that we need to update the constraint for playtime_entries, since a given patron shouldn't be "reading" the same book both with and without a loan_identifier within the already-existing constraints (`tracking_id`/timestamp, `identifier_str`, etc.).

That said, there's probably no harm in it (other than increasing the size of the index, of course).

Reply:
Maybe we should leave it in there as a sanity check?  I don't know. 

I defer to your judgement on this one,  not only because I respect your opinion but also because you are the Playtime Tracking Man!!! :)